release-notes.md

```diff
@@ -48,9 +48,9 @@ Although every attempt has been made to make this information as accurate as pos
 
 | Components | Version Numbers |
 | --- | --- |
-| indy-plenum | 1.6.52 |
+| indy-plenum | 1.6.53 |
```

Comment:
The version of Sovrin needs to be 1.1.30, not 1.1.26

Reply:
This has been added per the request.